src/nvim/testdir/test_startup.vim

```diff
@@ -559,3 +560,15 @@ func Test_zzz_startinsert()
   endif
   call delete('Xtestout')
 endfunc
+
+func Test_start_with_tabs()
+  if !CanRunVimInTerminal()
+    return
+  endif
+
+  let buf = RunVimInTerminal('-p a b c', {})
+  call VerifyScreenDump(buf, 'Test_start_with_tabs', {})
```

Comment:
could make a Lua screen test pretty easily (`startup_spec.lua`):

```lua
clear({args={'-p', 'a', 'b', 'c'})
screen:snapshot_util()  -- <= this will print a screen test, just paste it into the test.
```